python_modules/dagster/dagster/_core/execution/asset_backfill.py

```diff
@@ -1263,28 +1249,24 @@ def execute_asset_backfill_iteration_inner(
         failed_and_downstream_subset = AssetGraphSubset()
         next_latest_storage_id = instance_queryer.instance.event_log_storage.get_maximum_record_id()
     else:
-        updated_materialized_subset_data = None
-        for (
-            updated_materialized_subset_data
-        ) in get_asset_backfill_iteration_materialized_partitions(
+        next_latest_storage_id = instance_queryer.instance.event_log_storage.get_maximum_record_id()
+        cursor_delay_time = int(os.getenv("ASSET_BACKFILL_CURSOR_DELAY_TIME", "0"))
```

Comment:
default to 0 seconds delay if env var is not set to prevent tests from being very slow

Reply:
switched the default for this back to 0 (contradicting my own earlier feedback) while we're still figuring out the exact value here for different situations